src/api/latest-fid-pulls.ts

```diff
@@ -0,0 +1,48 @@
+import { LatestFidPullsRow } from '../db/db.types.js'
+import { db } from '../db/kysely.js'
+
+import { log } from '../lib/logger.js'
+
+const tableName = 'latest_fid_pulls'
+export async function upsertLatestFidPull(
+  fid: number,
+  updatedAt: Date
+): Promise<void> {
+  try {
+    await db.insertInto(tableName).values({ fid, updatedAt }).execute()
+
+    log.debug(`LATEST FID PULL UPSERTED FOR FID ${fid}`)
+  } catch (error) {
+    log.error(error, 'ERROR INSERTING LATEST FID PULL')
+  }
+}
+
+export async function selectAllLatestFidPulls(): Promise<
```

Comment:
I know this only gets called once when backfill starts, but it's possible that this returns hundreds of thousands of rows... Not sure if that will cause any postgres issues on a remote instance